stream_alert_cli/test.py

```diff
@@ -53,6 +53,33 @@ def report_output(cols, failed):
 
     print '\t{}\ttest ({}): {}'.format(status, *cols)
 
+def report_output_summary(rules_fail_pass):
+    """Helper function to print the summary results of all tests
+    Args:
+        rules_fail_pass [list]: A list containing two lists for failed and passed
+            rule tests. The sublists contain tuples made up of: (rule_name, rule_description)
+    """
+    failed_tests = len(rules_fail_pass[0])
```

Comment:
@jacknagz made some changes to support warnings. PTAL. output will be similar to:
<img width="676" alt="screen shot 2017-05-03 at 8 51 25 pm" src="https://cloud.githubusercontent.com/assets/13773789/25690059/56c2a608-3042-11e7-9e8d-2d9d43dbe99f.png">